pom.xml

```diff
@@ -26,11 +26,19 @@
             <artifactId>webdrivermanager</artifactId>
             <version>5.3.1</version>
         </dependency>
-        <dependency>
-            <groupId>org.seleniumhq.selenium</groupId>
-            <artifactId>selenium-java</artifactId>
-            <version>4.7.2</version>
-        </dependency>
+<!--        <dependency>-->
```

Comment:
These changes are not ok. You only needed to remove lines 34 to 41 from this pom file ->
![image](https://user-images.githubusercontent.com/18215854/214046806-4fb46035-3c38-4885-a720-4e83204cdf6f.png)